dolweb/downloads/templates/downloads-index.html

```diff
@@ -81,10 +80,11 @@ <h1>{% trans "Development versions" %}</h1>
 <div id="download-stable">
 <h1>{% trans "Stable versions" %}</h2>
 
-<div class="alert alert-info">{% blocktrans %}
-    Stable versions are released after a lot of testing to ensure emulation
-    performance and stability. However, since they are released less often,
-    they might be outdated and lacking some new features.
+<div class="alert alert-danger">{% blocktrans %}
+    The stable versions below are years out of date and missing countless features and bug fixes.
```

Comment:
Worth noting that we'll need to update the wording after a 6.0 release.

Reply:
I added a TODO to that effect.